hack/test-external.sh

```diff
@@ -13,8 +13,9 @@ time kapp delete -y -a istio
 time kapp deploy -y -a cert-manager -f examples/cert-manager-v1.6.1/
 time kapp delete -y -a cert-manager
 
-time kapp deploy -y -a knative -f examples/knative-v1.1.0/
-time kapp delete -y -a knative
+# TODO Add knative - Commenting because it need > resources than what github action provides.
```

Comment:
Question: Does it need more RAM and CPU?

Reply:
Yes.